### PR TITLE
HOPSFS-158: Addendum. Fixed compilation error

### PR DESCRIPTION
**Pull request by smkniazi**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added and passed (for bug fixes / features)
- [ ] HOPS JIRA issue has been opened for this PR
- [ ] All commits have been squashed down to a single commit
- [ ] The commit message has the following format: [HOPS-XXX] message

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/LeaseManager.java [hunk 1]
```diff
@@ -583,11 +583,7 @@ public void acquireLock(TransactionLocks locks) throws IOException {
                 .add(lf.getLeaseLockAllPaths(LockType.WRITE, holder,
                         fsnamesystem.getLeaseCreationLockRows()))
                 .add(lf.getLeasePathLock(leasePaths.size()))
-                .add(lf.getBlockLock()).add(lf.getBlockRelated(BLK.RE, BLK.CR, BLK.IV, BLK.ER,
-                    BLK.UC, BLK.PE, BLK.UR));
-            if (fsnamesystem.isCloudEnabled()) {
-              locks.add(lf.getProvidedCacheLocLoc());
-            }
+                .add(lf.getBlockLock()).add(lf.getBlockRelated(BLK.RE, BLK.CR, BLK.ER, BLK.UC, BLK.UR));
```
Comment (maismail) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/LeaseManager.java:586]: actually it seems to be missing BLK.IV and BLK.PE. Is that intentional?
```
BLK.RE, BLK.CR, BLK.IV, BLK.ER,
                    BLK.UC, BLK.PE, BLK.UR
```

Comment (smkniazi) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/LeaseManager.java:586]: compare it with this https://github.com/hopshadoop/hops/commit/2a07908f60944ac714b033e96d4bdfbe44ea88cf

Comment (smkniazi) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/LeaseManager.java:586]: IV and PE are from the enterprise branch that got added by mistake

Comment (maismail) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/LeaseManager.java:586]: aha, got it, thanks!

**Pull request merged**